pkg/cluster/sync_test.go

```diff
@@ -0,0 +1,212 @@
+package cluster
+
+import (
+	"fmt"
+	"testing"
+
+	acidv1 "github.com/zalando/postgres-operator/pkg/apis/acid.zalan.do/v1"
+	"github.com/zalando/postgres-operator/pkg/util/config"
+	"github.com/zalando/postgres-operator/pkg/util/k8sutil"
+
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func int32ToPointer(value int32) *int32 {
```

Comment:
Is it a  duplicate of [pkg/controller/operator_config.go:func int32ToPointer](https://github.com/zalando/postgres-operator/pull/799/files#diff-ab4a7db75ac2e59f513f47a9d540726dR25) ?